src/types.ts

```diff
@@ -70,7 +76,7 @@ export interface BlockRange {
   last: number
 }
 
-export interface ConfirmedTransaction {
```

Comment:
`export type ConfirmedTransaction = Transaction` to remain backwards compatible.